### ISSUE TITLE
CUPS user `print` has no password set as `mkpasswd` is missing

### ISSUE THREAD

**Issue by MaxWinterstein** [zajac-grzegorz/homeassistant-addon-cups-airprint#6]
```bash
docker build -t foo .
...
 docker run --rm -ti --entrypoint /bin/bash foo
root@8ba099d1a4b8:/# mkpasswd
bash: mkpasswd: command not found
root@8ba099d1a4b8:/# cat /etc/passwd | grep print
print:x:1000:1000::/home/print:/bin/bash
```

```
root@8ba099d1a4b8:/# apt update
...
root@8ba099d1a4b8:/# apt install whois
...
root@8ba099d1a4b8:/# mkpasswd
Password:
$y$j9T$qkGeygtaoLpHoNN6vN0CU0$eEab58FX7R8QimvHKzoQeOBvYJkOUpE6/voQqAk9w66
root@8ba099d1a4b8:/# which mkpasswd
/usr/bin/mkpasswd
```

Sadly `command not found` is not raised through the subshell / argument and docker does not abort.

Proposal to fix: #5

**Comment by zajac-grzegorz** [comment]
thanks!

**Comment by MaxWinterstein** [comment]
Thanks for your work 👍 

Fixed via #5 
/close
